internal/orchestrator/restore_decision_test.go

```
@@ -62,6 +62,39 @@ func tarBytes(t *testing.T, files map[string]string) []byte {
 	return buf.Bytes()
 }
 
+func TestReadRestoreDecisionMetadataAcceptsTypeRegA(t *testing.T) {
+	data := []byte("BACKUP_TYPE=pbs\n")
+	var buf bytes.Buffer
+	tw := tar.NewWriter(&buf)
+	if err := tw.WriteHeader(&tar.Header{
+		Name:     "var/lib/proxsave-info/backup_metadata.txt",
+		Typeflag: tar.TypeRegA,
```

golangci-lint (2.12.1)

[error] SA1019: tar.TypeRegA has been deprecated since Go 1.11 and an alternative has been available since Go 1.1: Use TypeReg instead. 

(staticcheck)